### PR TITLE
feat: new protobuf files and match latest version of chain

### PR DESCRIPTION
**Pull request by gluax**
## Motivation

Adds the batching and private key modules.

## Explanation of Changes

The two new modules are added. As well as the oracle program renames.

## Testing

Works with the test suite :)

## Related PRs and Issues

N/A

### CODE REVIEW

File: crates/proto-common/proto/sedachain/batching/v1/batching.proto [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  // the validator tree.
+  repeated bytes validator_entries = 3;
+}
+
+// BatchSignatures contains basic validator data and its batch signatures
+// under various cryptographic schemes.
+message BatchSignatures {
+  string validator_addr = 1
+      [ (cosmos_proto.scalar) = "cosmos.ValidatorAddressString" ];
+  int64 voting_power = 2;
```
Comment (hacheigriega) [inline crates/proto-common/proto/sedachain/batching/v1/batching.proto]: This is being removed in `hy/batch-skipping`

Comment (gluax) [reply crates/proto-common/proto/sedachain/batching/v1/batching.proto]: I'll wait for that to merge on chain side before merging this then ^^

**Comment by Thomasvdam** [comment]
Maybe good to use the proto files from the open PR on the chain side? Minor change to the pubkey module.

**Comment by gluax** [comment]
> Maybe good to use the proto files from the open PR on the chain side? Minor change to the pubkey module.

Ah, I might wait on that, though, just because I don't know if it will get merged in and deployed by Monday, and I don't want to break Mario and Franklin's working with Planet.

**Review by Thomasvdam — APPROVED** [review]
It shouldn't matter since those files are for modules we don't interact with, but by that same logic we can just merge this as is and wait for the next update that does actually change RPC calls we care about :D

**Pull request merged**